core/domain/takeout_domain_jobs_one_off.py

```diff
@@ -78,3 +78,44 @@ def reduce(key, values):
             yield (key, len(values))
         else:
             yield (key, values)
+
+
+class SnapshotMetadataCommitMsgMigrationOneOffJob(
+        jobs.BaseMapReduceOneOffJobManager):
+    """Job that indexes the commit_message field of the
+    BaseSnapshotMetadataModels.
+    """
+
+    @classmethod
+    def enqueue(cls, job_id, additional_job_params=None):
+        super(SnapshotMetadataCommitMsgMigrationOneOffJob, cls).enqueue(
+            job_id, shard_count=8)
+
+    @classmethod
+    def entity_classes_to_map_over(cls):
+        return [
+            config_models.ConfigPropertySnapshotMetadataModel,
+            collection_models.CollectionRightsSnapshotMetadataModel,
+            collection_models.CollectionSnapshotMetadataModel,
+            exploration_models.ExplorationRightsSnapshotMetadataModel,
+            exploration_models.ExplorationSnapshotMetadataModel,
+            skill_models.SkillSnapshotMetadataModel,
+            story_models.StorySnapshotMetadataModel,
+            topic_models.SubtopicPageSnapshotMetadataModel,
+            topic_models.TopicRightsSnapshotMetadataModel,
+            topic_models.TopicSnapshotMetadataModel,
+            question_models.QuestionSnapshotMetadataModel,
+        ]
+
+    @staticmethod
+    def map(item):
+        try:
+            item.put(update_last_updated_time=False)
+        except Exception as e:
+            yield ('FAILURE' + e.strerror, 1)
```

Comment:
'FAILURE %s' % e.strerror

Also provide the item ID, item type, etc. -- i.e. enough info to identify the item.

Reply:
Done